spring-boot-jib/src/main/resources/logback.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file necessary for Spring Boot to work, or is it just a good practice? [The documentation](https://docs.spring.io/spring-boot/docs/current/reference/html/howto-logging.html) seems to suggest that we don't need this.

If we don't need it, then lets remove it, just so the focus of the example is on jib.

Reply:
Yes, its not needed. Anyways, any failure to build using Jib is going to be logged at console, so I will remove it. Thanks

Reply:
Removed the file